test/com/jambit/DatabaseConnectionTest.java

```diff
@@ -0,0 +1,70 @@
+package com.jambit;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.jambit.domain.MoodEntries;
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+
+/*
+TODO:
+   - More Tests
+   - Delete Database content after all tests (@Afterall)
+   - Implement H2 (optional)
+ */
+
+class DatabaseConnectionTest {
+  private static DatabaseConnection databaseConnection;
+
+  @BeforeAll
+  static void init() throws IOException, SQLException {
+    DatabaseConnection.changePropertyPath("config/appTest.properties");
+    databaseConnection = DatabaseConnection.getInstance();
+  }
+
+  @Test
+  @Order(1)
+  public void insertTest() throws SQLException {
```

Comment:
https://osherove.com/blog/2005/4/3/naming-standards-for-unit-tests.html
Think about a good and proper name...